site/content/docs/04-preloading.md

```diff
@@ -132,3 +132,23 @@ You can abort rendering and redirect to a different location with `this.redirect
 	}
 </script>
 ```
+
+#### Typing the function
+
+If you use TypeScript and want to access the above context methods, TypeScript will thow an error and tell you that it does not know the type of `this`. To fix it, you need to specify the type of `this` (see the [official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#specifying-the-type-of-this-for-functions)). We provide you with helper interfaces so you can type the function like this:
+
+```html
+<script context="module">
+	import type { Page, PreloadContext } from "@sapper/common";
```

Comment:
Sorry to comment on a merged PR, but both variables doesn't seem to be used below, shouldn't it be `Preload`?

Reply:
Good catch, yes this is wrong, it should be `Preload`.

Reply:
Fixed on master